### PR TITLE
Make it docker-composable for now

### PR DESCRIPTION
**Pull request by sh19910711**

### CODE REVIEW

File: mgmt/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+RUN git clone --depth 1 --recursive https://github.com/codestand/mgmt /app
+WORKDIR /app
+RUN gem install bundler
+RUN bundle install -j4 --without development test mysql
+# TODO: update config/database.yml
+RUN echo 'production:' > config/database.yml && \
+    echo '  url: <%= ENV["DATABASE_URL"] %>' >> config/database.yml
+RUN echo 'production:' > config/secrets.yml && \
+    echo "  secret_key_base: \"$(cat /dev/urandom | strings | tr -dc 'a-zA-Z0-9' | head -c 128)\"" >> config/secrets.yml
+RUN cat config/secrets.yml
```
Comment (seiyanuta) [inline mgmt/Dockerfile]: I think it doesn't make sense.

File: gateway/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,26 @@
+FROM alpine:3.4
+
+RUN apk update && apk upgrade && \
+  apk add --virtual builddeps gcc g++ make curl ruby-dev ruby-rake bison perl git && \
+  apk add pcre-dev zlib-dev
+
+ENV NGINX_VERSION="1.10.1"
+
+# build nginx
```
Comment (seiyanuta) [inline gateway/Dockerfile]: Why do you build nginx from source?

File: mgmt/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,20 @@
+FROM ruby:2.3.1-alpine
+
+# middleware
+RUN apk update && apk upgrade && \
+  apk add --no-cache git build-base libxml2-dev postgresql-dev supervisor nodejs tzdata sqlite-dev && \
+  mkdir /var/run/supervisor && mkdir /var/log/supervisor
+
+# webapp
+RUN addgroup -S web && adduser -G web -S web && \
```
Comment (seiyanuta) [inline mgmt/Dockerfile]: Why do you do those things at once?

File: emu-server/Dockerfile [hunk 4]
```diff
@@ -1,27 +0,0 @@
-FROM ubuntu:16.04
```
Comment (seiyanuta) [inline emu-server/Dockerfile]: What does `🙇` emoji in the commit message mean?

File: .travis.yml [hunk 5]
```diff
@@ -0,0 +1,15 @@
+language: bash
+sudo: required
+services:
+  - docker
+
+install:
+  - docker-compose build
+  - docker-compose run mgmt bundle config build.nokogiri --use-system-libraries
+  - docker-compose run mgmt bundle install --jobs 4 --without test --without development --without mysql
```
Comment (seiyanuta) [inline .travis.yml]: Do this in the Dockerfile.

File: emu-server/Dockerfile (hunk 4 above)
Comment (sh19910711) [inline emu-server/Dockerfile]: It means "Thanks for working it" ;-)

Comment (seiyanuta) [inline emu-server/Dockerfile]: It looks like *apology*. Please remove it.

**Comment by seiyanuta** [comment]
Looks good ✨

**Pull request merged**